anitab-org/bridge-in-tech-web
# Feat view organizations list and other organization profile

### Description
Added View Organizations list and Other Organization Profile functionalities

Fixes #52 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- Login and go to Organizations page. see list of organizations
<img width="1653" alt="Screen Shot 2020-08-18 at 12 07 10 am" src="https://user-images.githubusercontent.com/29667122/90405244-d80c3500-e0e6-11ea-9a13-c9f3a4460bd7.png">
- Click on an organization from the list and see that Organization Profile page
<img width="1605" alt="Screen Shot 2020-08-18 at 9 46 56 am" src="https://user-images.githubusercontent.com/29667122/90455424-5e516700-e139-11ea-8eb4-e09bd5e572e8.png">
**Note**
Notice the different Navbar options when you login as user who's representing organization or not.

As a represenative of an organization, you'll see the Navbar below:
<img width="1668" alt="Screen Shot 2020-08-18 at 10 01 46 am" src="https://user-images.githubusercontent.com/29667122/90455629-e5064400-e139-11ea-9c1c-8f286784f296.png">

Whereas if you don't represent an organization, you'll see the below Navbar:
<img width="1669" alt="Screen Shot 2020-08-18 at 10 01 26 am" src="https://user-images.githubusercontent.com/29667122/90455669-ff402200-e139-11ea-9800-e4c8fcdef9be.png">




### Checklist:


**Code/Quality Assurance Only**

**Additional Note**
To test this features, you should only use the BIT backend server from branch [PR#112](https://github.com/anitab-org/bridge-in-tech-backend/pull/112) as well as MS backend server for BIT ([ms-backend-server](https://github.com/mtreacy002/mentorship-backend/tree/ms-backend-server))
